index.js

```diff
@@ -26,21 +25,18 @@
    * @constructor
    */
   function FastDom() {
-    this.frames = [];
-    this.lastId = 0;
-
     // Placing the rAF method
     // on the instance allows
     // us to replace it with
     // a stub for testing.
-    this.raf = raf;
+    this.raf = raf.bind(window);
```

Comment:
Native RAF wants to be called on `window` and I've used `this.raf`. I suppose I could assign it to a temp variable first.